bukuserver/views.py

```diff
@@ -209,11 +209,13 @@ def create_form(self, obj=None):
             form.url.data = request.args.get('link', form.url.data)
             form.title.data = request.args.get('title', form.title.data)
             form.description.data = request.args.get('description', form.description.data)
+            form.tags.data = request.args.get('tags', form.tags.data)
+            form.fetch.data = request.args.get('fetch', request.data.get('fetch', True))
```

Comment:
This is invoked in both GET request when fetching the form, and POST request when submitting it.
In either case, if the `fetch=` parameter is present it will be used, otherwise falling back to default value.